src/lib/components/ui/Search.svelte

```
@@ -0,0 +1,179 @@
+	import Input from './input/input.svelte';
+
+	const client = algoliasearch('99IEWD8Z0K', 'c6f824db1a70a8523780908459090a48');
+	// const client = algoliasearch(
+	// 	import.meta.env.VITE_ALGOLIA_APP_ID,
+	// 	import.meta.env.VITE_ALGOLIA_SEARCH_KEY
+	// );
+
+	type SearchResult = {
+		objectID: string;
+		title: string;
+		_highlightResult?: HighlightResult & { content_excerpt: any };
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)